# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaEngineWebRTCAudio.h
# Commit: b004b579d1ce
# Full Hash: b004b579d1ce8fa91acc2330c436340b099ceccd
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Rename functions r=pehrsons
#   
#   Rename {Close, Open}AudioInput to {Disc, C}onnectDeviceInput
#   
#   Depends on D134533
# ==============================================================================

diff -r 8cb7b8231c98 -r b004b579d1ce dom/media/webrtc/MediaEngineWebRTCAudio.h
--- a/dom/media/webrtc/MediaEngineWebRTCAudio.h	Thu Jan 27 22:29:35 2022 +0000
+++ b/dom/media/webrtc/MediaEngineWebRTCAudio.h	Thu Jan 27 22:29:35 2022 +0000
@@ -226,7 +226,7 @@
   RefPtr<MediaInputPort> mPort;
 
   // Only accessed on the main thread. Used for bookkeeping on main thread, such
-  // that CloseAudioInput can be idempotent.
+  // that DisconnectDeviceInput can be idempotent.
   // XXX Should really be a CubebUtils::AudioDeviceID, but they aren't
   // copyable (opaque pointers)
   RefPtr<AudioDataListener> mInputListener;
@@ -245,10 +245,10 @@
   // Users of audio inputs go through the track so it can track when the
   // last track referencing an input goes away, so it can close the cubeb
   // input. Main thread only.
-  nsresult OpenAudioInput(CubebUtils::AudioDeviceID aId,
-                          AudioDataListener* aListener,
-                          const PrincipalHandle& aPrincipal);
-  void CloseAudioInput();
+  nsresult ConnectDeviceInput(CubebUtils::AudioDeviceID aId,
+                              AudioDataListener* aListener,
+                              const PrincipalHandle& aPrincipal);
+  void DisconnectDeviceInput();
   Maybe<CubebUtils::AudioDeviceID> DeviceId() const;
   void Destroy() override;
   void SetInputProcessing(RefPtr<AudioInputProcessing> aInputProcessing);
